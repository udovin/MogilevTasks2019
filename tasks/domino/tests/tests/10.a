1 1 2 2 1 1 1 1 1 2 1 1 1 1 2 574 1 1 1 2 1 1 2 1 25 1 1 3 1 1 1 1 1 1 1 2 1 1 222 1 1 1 1 1 2 1 1 1 2 1 2 1 1 1 2 1 2 1 1 2 1 1 2 2 1 1 1 3 1 3 2 1 1 1 1 3 1 1 1 1 1 1 2 1 1 1 1 526 1 1 1 1 1 1 1 1 2 1 2 1 2 1 1 1 1 1 1 1 1 2 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 2 1 1 2 1 1 304 1 2 1 1 2 3 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 3 1 1 1 1 2 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 1 801 2 1 1 2 1 1 1 1 2 1 1 1 1 176 1 1 1 1 1 4 1 1 1 1 1 3 1 1 1 1 1 2 133 1 2 1 1 2 2 1 1 2 1 1 1 1 1 1 1 769 1 2 367 1 1 1 1 2 1 106 1 1 1 1 1 1 1 1 150 1 1 2 4 781 1 1 148 1 1 2 1 3 2 1 1 1 1 2 1 2 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 3 1 1 3 1 1 1 3 1 1 1 2 109 1 1 1 1 1 1 2 2 1 1 2 4 1 1 1 1 3 46 2 1 1 1 1 6 2 3 1 1 332 1 1 556 2 1 1 2 1 1 2 1 759 1 2 2 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 2 1 1 1 2 1 1 1 3 1 1 2 1 1 1 1 1 1 3 1 1 1 2 1 1 2 1 1 1 1 2 1 2 1 1 1 1 1 1 1 1 2 1 1 1 1 621 2 1 724 2 2 2 1 4 651 1 1 1 2 1 617 1 1 495 1 1 1 2 1 1 1 1 1 1 1 2 1 3 1 1 1 2 1 1 2 1 1 1 1 2 1 1 3 3 2 2 1 2 1 1 1 1 1 1 1 685 1 1 1 1 1 1 1 1 3 371 1 1 1 1 2 3 2 1 1 3 1 1 1 1 1 1 4 2 1 1 1 144 3 3 3 1 2 1 1 2 2 1 3 1 1 2 3 1 3 1 1 2 1 1 3 2 1 1 1 305 348 1 1 1 1 2 1 2 1 1 1 2 1 1 1 1 2 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 4 1 1 1 2 1 1 2 1 1 1 1 2 1 1 2 328 2 1 1 1 1 1 3 1 1 1 1 1 1 1 1 2 251 1 1 3 1 1 1 1 1 1 2 1 2 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 49 3 1 3 2 1 1 2 1 3 1 2 1 2 1 3 490 1 3 2 1 2 1 1 1 3 3 780 2 2 1 2 2 1 2 1 1 1 1 1 1 1 1 1 1 1 2 1 3 1 1 1 1 1 1 1 1 2 1 1 1 2 1 2 1 1 1 1 1 3 1 1 1 1 1 1 2 1 293 1 1 2 2 1 1 1 1 1 2 1 1 1 1 1 1 1 3 1 1 1 1 1 1 1 1 2 1 2 2 1 3 4 1 1 1 1 2 1 1 1 1 1 1 1 2 1 3 1 1 2 1 1 1 2 1 1 1 1 1 1 1 784 2 1 1 1 4 1 341 1 1 1 
