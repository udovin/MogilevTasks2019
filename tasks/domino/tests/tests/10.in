801
-33149754 29619
-38747456 33279
-16539835 45050
-35768679 38489
-44114776 25529
-38394281 18357
-32753398 3187
-33492620 30059
-24751905 22310
-42446382 26658
-35728966 4465
-26534240 1293
-31497375 6585
-30939930 3916
-47313294 35642
-38220486 77811598
-35554843 1389
-26033624 704
-35386008 25437
-33873066 59146
-41518278 16418
-41334754 4012
-18742188 6814
-44385027 6142
-14955438 41995284
-32359668 29027
-36347547 23849
-31891865 41708
-22262610 26241
-42814952 17449
-23945142 14391
-45941411 17458
-45303168 18454
-42185126 17456
-14230882 3589
-26718167 63664
-43019444 44060
-40734013 3493
-23316452 77971074
-24528152 6105
-14946102 5696
-36878733 13892
-15651948 15219
-41301112 6924
-15386319 35089
-32022054 29830
-17194732 6408
-36165612 3097
-32682274 65154
-31307178 1833
-27795895 10409
-40109536 1536
-34530628 10124
-26393072 17313
-35475118 73502
-38827844 20670
-35556434 16223
-36733579 2542
-20018412 17645
-24783031 32937
-18083097 882
-34370050 1947
-29623561 42922
-42671608 20941
-46572646 30899
-23558299 12075
-31018937 6411
-39470818 46775
-33945573 5764
-22966222 42495
-18043301 6672
-47436112 7789
-36856909 13678
-35138790 7045
-47561835 17425
-23041709 31761
-45008778 6698
-23032850 4985
-35937410 6416
-24544720 289
-20288102 13273
-36569124 2390
-46115013 35590
-42271195 23632
-48045412 10190
-16279364 8491
-28970371 2409
-36102439 95625643
-43339565 21910
-21547832 4017
-15356410 2849
-17187270 4426
-28945982 7916
-16964230 35091
-16594252 16522
-44302215 31289
-20512439 17011
-47427281 4130
-26537341 22485
-43649640 1899
-40579869 52086
-16198430 49457
-41392074 292
-15448631 16911
-35578735 872
-44735566 40610
-33459764 19759
-43675617 1044
-16332869 9461
-29139540 36632
-28922836 5482
-42527406 8301
-28722334 7152
-31788935 15614
-39691779 59292
-41936852 33223
-18740477 1607
-19956837 23651
-34825829 15758
-25128194 1243
-14070284 12181
-23968035 1466
-26899042 1641
-24831726 47689
-46633478 2173
-21751502 38864
-23301105 94
-34231334 25848
-20692678 2891
-47494099 32648
-14225616 443
-15541395 20861
-46819795 198
-30282989 4674
-23635024 26471
-36642385 18539
-39459753 1355
-26561738 42139073
-23781992 14480
-36171655 37880
-26309259 135
-46232465 25062
-38325735 66889
-24561045 69062
-26880223 10241
-25346869 16726
-19481051 659
-25052182 802
-41093492 16800
-16486754 4265
-44165425 27850
-17614217 6520
-29394828 15276
-47312568 73
-23288727 1483
-46998355 37298
-17053157 5634
-31958596 306
-23191483 9172
-26437620 46265
-23040737 275
-45676919 6237
-29130609 6442
-27048089 35419
-47086503 31235
-16433974 42811
-43842363 50270
-46380029 29616
-40094152 4530
-14277309 38483
-30388355 20661
-41038234 29300
-36927020 26332
-36766947 2294
-40045432 34603
-20386382 23946
-17457472 5839
-22566801 34872
-42894314 14650
-24366666 3033
-21438069 12361
-45152958 22593
-48088790 86438498
-41793239 4371
-25623867 4353
-34903599 27112
-29961398 36653
-26421175 9943
-15733550 53960
-18313729 5679
-22385402 33755
-40746093 14440
-16898308 14379
-34711498 14374
-46791903 3258
-43544202 6468
-21347032 61107284
-23489408 17657
-17878661 5762
-39765838 33027
-38681719 32802
-35340335 17758
-23998074 40292
-19312850 6093
-43089834 3699
-45281115 3012
-24640048 19791
-35752695 15119
-33624905 63571
-25522560 12266
-38046874 17004
-39370698 12532
-32284818 16416
-46453821 15469
-26113829 31070
-19495255 21217213
-29740789 24133
-14589907 29465
-41789197 373
-44907201 247
-41398072 10287
-15593274 48187
-35673252 13083
-33004475 7066
-25413705 45640
-38100353 12507
-33037778 14806
-29527635 12208
-20340445 42144
-14572482 7323
-29204062 15068
-43501344 9803
-46701305 91999161
-22649622 2684
-40509613 12216
-29304375 83502540
-23597320 3914
-32603329 26767
-46341977 2563
-30325616 25071
-17661181 7520
-43610607 18335
-18319875 7754867
-16314865 4456
-19263673 33602
-26800880 10894
-34039857 1385
-37839446 39081
-43970157 3322
-30579300 33403
-27814660 15820
-20227061 49131066
-18703894 5502
-36780486 12002
-44006433 29412
-44425099 60740
-47232246 89981331
-40889983 23897
-23395374 10285
-20116099 34669824
-16857688 21628
-21252603 9860
-19412079 47829
-45546830 6927
-17679452 24806
-30897925 20401
-44655149 31150
-28962882 3209
-27197005 3576
-17915763 23698
-20991811 11263
-15485904 31473
-33567077 18758
-38935900 20357
-19161529 6473
-24468576 2790
-27369824 11397
-32135141 8170
-39434272 21041
-16008247 622
-35149421 4925
-34386223 45451
-14159052 1400
-21043275 15573
-31397775 40050
-18922518 3387
-30270023 8476
-35066056 24560
-23428113 49052
-30070539 495
-46954284 9093
-27217081 34827
-17657807 2650
-20735191 1876
-45967172 16349
-18856103 23509
-25466338 9779
-33551135 6460
-40457123 12418
-17319456 13691
-18504556 30102167
-18157277 37820
-22787006 10573
-29228057 6867
-33976438 19592
-16322574 7564
-39609868 30163
-33999353 23127
-39800558 61489
-40966261 9542
-14335068 9464
-43382165 43393
-34105792 62387
-42432900 3455
-33221784 11647
-43160448 6053
-38637817 25930
-41178508 71295
-15911982 22980851
-22688148 38538
-21117854 28982
-33695712 6456
-26691386 2121
-34980184 33450
-33385258 301711
-15788715 75781
-19803619 38275
-20664726 7468
-16137084 13903
-27732128 57406289
-24676705 15219
-46763062 20723
-37323863 13504863
-32767675 70227
-45403049 11895
-44404143 3492
-23902663 41011
-41361147 10155
-21633726 14725
-17463519 12586
-42944840 1624
-46175985 21657331
-34043326 304
-46023304 77437
-44907534 28991
-14105566 15588
-45079146 43026
-18239576 19315
-41157429 4933
-27791344 2930
-18037372 3410
-20504304 1865
-28339566 19155
-42569591 78492
-47936604 4125
-30689613 4592
-20812521 62705
-36095976 515
-39324963 23994
-38563764 26104
-27841599 41127
-30895449 1704
-32121000 10131
-34782018 11267
-26331306 28042
-17011083 5477
-31707779 22240
-34289617 45475
-37078692 58715
-21961786 709
-30741248 27217
-42211966 51975
-14622078 6925
-31439311 33270
-32955491 20168
-38907305 8423
-35310505 22160
-36224291 5502
-35160117 44907
-30175809 5791
-23823165 20572
-16384806 789
-28989985 22445
-25943196 6095
-28863803 44916
-42062555 47388
-25293615 41514
-27918358 23967
-14501897 3666
-44422344 836
-30471239 33972
-37521946 18074
-21655514 38961
-17061714 502
-22296239 18103
-20541813 20214
-38197207 2659
-25181284 4094
-24295234 33879
-36319712 25613
-30453154 15109
-44503390 34152
-18400783 5738
-25830701 23311
-20456369 27084
-17762812 5608
-40381339 27088018
-21565243 51613
-32392284 13074
-44579030 17360912
-37147663 7123
-35959683 35822
-17886593 25526
-41144986 11097
-27259438 43179
-41606655 8598137
-32682227 22
-40506317 1085
-45468973 9912
-23974750 8220
-18817947 11580
-40212737 99365633
-15592904 145
-32849600 23847
-34854960 1623524
-20054713 29647
-31122210 11852
-25372826 15419
-25638128 13414
-44496144 907
-45817284 1317
-41653897 28514
-41848914 4696
-28276122 3733
-39929354 903
-25695942 26572
-19009578 57439
-47707616 30084
-35832435 65900
-37461335 10275
-44969481 1917
-23716670 9725
-26036520 30893
-15159325 3150
-44350730 10360
-19768485 27039
-46104685 5241
-38285903 30800
-14384160 1251
-39212131 911
-28367017 39482
-37616964 46266
-15302174 7902
-18056150 26110
-25186185 45761
-27691287 20599
-22923790 30370
-27685809 4262
-41854131 8832
-38074735 19046
-14870355 6616
-44812259 8715
-28449435 41281
-28602650 15852
-14890966 8328
-27457934 35829
-42974434 15204239
-43097817 5305
-43990833 9759
-28210742 13991
-20991533 216
-42779167 15027
-18839420 2361
-37717742 7214
-45252320 18738
-40776440 40288
-29464764 55922397
-28542867 10914
-30807451 513
-46525437 42383
-31249542 5825
-30961354 48730
-41798566 22137
-30808017 3646
-34556821 1490
-45519875 24441
-40315297 29040
-35022849 15391
-42483734 26858
-21426455 2312
-27185195 2275
-24951206 26484
-26503672 7664
-36575005 53837
-31540559 58225
-32318899 17097
-16785041 24547
-31055821 11716
-19906855 84431148
-41976786 40476
-21454140 46644
-30099653 40013
-29610169 4209
-14819697 43885
-29058354 3998
-44541361 9902
-43181301 49232
-28732348 35310
-14690266 38217
-26931302 46837
-39284004 3621
-24424427 5001
-40291455 4497
-40141485 40299
-21994646 17031
-41877372 27138
-29394596 132
-21865094 54810
-24070780 34533
-19654326 1084
-15869428 24266
-45838298 49573
-42946484 4270
-16092296 34973
-20578027 3225
-32181793 15766
-26624901 64713
-28493488 90785763
-28629504 7983
-38482272 48136
-14630081 6026
-17265775 22344
-45695306 44591
-39978398 4815
-40112785 17401
-17151974 35
-41570930 20164
-44854566 6733
-30842984 28619
-19392338 12972
-44206355 6206
-47640288 45374
-35910632 11393
-46637271 22049
-34435105 12572
-39072992 20479
-42973285 303
-15526751 9921
-17311136 1390
-39931126 38165
-14150081 3019
-31967986 2779
-16513640 1803
-22899708 17427
-47901622 188
-31890714 867
-47853545 23091
-42651606 9002
-17964810 16297
-32473536 57083
-43942993 9723
-17751675 2099
-37282046 12462
-43677670 245
-32203575 70665
-28117228 26832
-28250645 10086
-46830958 4446
-37540322 64701
-24169897 15317
-36085961 6348
-33242709 23067
-30064363 5793
-20183468 29168
-31642332 13698
-22055270 14254
-36376988 49322
-18102389 3694
-40631961 5213
-30626938 30678
-27539604 31411037
-30288269 13601
-37978899 610
-29509647 11123
-14451096 21673
-41261000 6929
-16669424 31968
-44220485 46104
-22118177 45131
-32058350 5226
-32891918 7846
-41909750 18882
-30533779 41669
-20904835 24381
-14809482 3503
-48010525 5189
-26902036 10141
-24372681 19111801
-22731914 35508
-23963829 686
-19657449 57226
-19078960 25380
-25770194 9762
-16702780 21879
-22828946 15727
-19133821 18836
-30617481 1865
-33075841 61047
-20877386 8578
-27384655 45887
-28073524 10780
-23384626 3838
-26088652 20997
-40550173 16762
-37790302 26900
-45758817 15099
-18557580 3077
-26365144 2005
-48074777 10846
-21013058 7195
-41317868 16464
-31318061 27766
-37213388 11685
-17127776 18813
-16614916 17122
-40289760 944
-30818097 2602
-47791786 13147
-27107823 493
-47042213 15115
-43784058 50494
-43436105 28827
-29783907 3916
-42135829 18135
-46271648 60275
-22177161 13184
-24142645 6141
-32151756 7302
-21178844 19383
-22421436 9130
-15075511 14272
-45222037 5788
-33855686 16033
-34225775 632
-31857066 3086
-47913966 6189
-36724440 5614
-15989957 12105
-18635012 51274
-16050291 17843781
-28641826 65725
-35232869 49097
-28782690 56634
-37023350 49116
-23899785 1208
-37698620 10591
-33779691 62209
-31772111 5840
-22433694 51080
-38978819 9169
-21797768 54904
-39145924 57131
-25548119 63536
-43916755 15245
-34058283 21518
-34576608 32832985
-33182547 28420
-17479830 50542
-23224464 41936
-27320273 13127
-19199426 31349
-41708631 30233
-17554222 33274
-36521188 28272
-41442988 51833
-43709347 41016
-47163014 47417301
-35101833 11299
-14513800 32314
-45888418 34839
-25961035 19250
-32412074 24879
-40209769 498
-26372990 14490
-37959179 1489
-38876977 693
-21947115 13328
-45176154 19620
-25630269 3378
-23078924 3110
-31602936 10639
-27618547 65984
-42744853 26729
-36550224 4802
-31087179 11321
-18426773 71277
-39034755 29770
-45328584 66405
-39537490 15459
-19743362 210
-44069766 8233
-24995969 1500
-25165660 979
-38891161 13841
-41289051 1831
-31167629 32342
-21200248 48772
-23617312 11786
-35091924 7280
-34186952 8267
-18841972 2860
-37908275 12013
-36255895 40251
-45832267 3461
-43258765 13039
-24241298 15874
-39246703 32744
-19548893 32503
-36798545 44335
-47073512 6660
-33735169 32762
-24040035 18916
-15147332 4831
-16814576 10131
-29356422 35382
-44868923 29654
-33318994 2884
-26252342 82715954
-17389782 48333
-18770045 3146
-40671946 56333
-23119243 65077
-36980699 11369
-38430330 34269
-19730048 9264
-30253408 1234
-14415884 6825
-27108675 28244
-23534641 5820
-36458590 12541
-19887465 16840
-42349852 25500
-19180952 15070
-27999044 55363
-18952700 19862
-17800154 56206
-36015124 25858
-22315160 698
-26866146 10470
-24890796 12278
-38176735 14255
-36485209 14344
-27003074 15503
-16429870 2521
-26817154 43610
-21512569 1920
-34459095 31512
-47901912 4854
-29778661 1096
-20583585 56397
-35598793 49676
-35446144 21728
-47748429 11408
-15200922 20373
-26985005 6320
-24191541 47991
-42031767 13996
-20958147 26124
-21706551 28758
-39883722 38031
-37145469 1775
-22493455 14382
-15269717 19289
-45605762 66424
-25244452 25225
-29666640 61450
-22231172 7220
-34639856 19501
-24486807 25192
-23686609 29988
-32555204 6681
-29864537 40216
-15094684 31094
-29947699 1833
-15037945 6583
-41739107 16826
-19867062 12591
-44199372 5385
-37391832 26882
-40852124 29234
-47353080 37895703
-25863961 61199
-38604911 12502
-23291924 1010
-26179653 65132
-21265526 72850
-14730901 33729
-28148159 45863912
-30033259 26801
-19610057 30144
-46898177 47279
