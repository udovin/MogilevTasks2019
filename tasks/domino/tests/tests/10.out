1 1 1 3 1 460 2 1 1 1 1 1 1 1 1 3 1 2 2 2 1 2 1 1 2 2 1 1 2 1 1 2 1 1 1 1 3 2 1 1 1 1 259 1 2 1 1 2 1 4 1 1 1 1 1 1 1 1 2 1 1 2 208 1 1 1 1 1 1 1 1 3 1 1 1 16 1 1 36 1 1 1 2 1 2 1 2 1 1 1 1 3 3 1 449 2 1 1 2 1 1 2 1 1 1 274 1 1 1 1 1 1 1 2 1 2 1 1 1 3 1 2 1 15 1 1 1 1 1 1 273 1 1 1 2 1 63 1 2 1 1 1 1 1 1 6 1 1 1 325 1 1 2 1 2 1 1 1 1 1 1 2 1 1 1 2 1 1 1 1 1 1 3 2 2 1 1 3 2 2 263 1 2 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 64 1 1 1 1 1 1 2 1 1 139 1 1 4 1 1 1 1 2 1 1 1 2 1 1 1 2 159 1 1 1 1 1 1 2 1 2 1 1 1 112 1 3 1 1 1 1 1 5 1 1 1 1 3 1 1 1 1 1 3 1 2 1 1 1 1 1 3 1 2 2 1 1 1 1 3 1 1 1 2 1 2 166 1 1 1 138 1 1 1 1 1 1 1 1 2 1 2 1 2 2 1 1 1 1 1 2 2 111 2 1 1 1 1 1 1 1 2 1 4 1 1 1 1 1 1 1 2 1 1 1 1 1 4 1 1 2 1 2 2 2 1 1 1 1 1 1 1 2 2 1 2 1 1 1 1 2 1 1 2 1 1 1 2 1 2 2 1 3 1 1 1 2 3 1 2 1 1 1 1 1 1 1 1 1 3 1 2 1 1 2 2 1 2 1 1 1 1 1 2 1 1 1 1 1 1 1 1 2 4 1 167 1 1 1 2 1 2 2 1 1 1 2 1 2 1 1 1 1 1 1 2 2 1 1 245 1 102 1 1 1 1 1 1 1 5 1 1 1 1 1 4 1 2 1 1 12 2 1 1 1 2 
